packages/flutter/test/material/autocomplete_test.dart

```diff
@@ -104,14 +104,17 @@ void main() {
     expect(list.semanticChildCount, 6);
   });
 
-  testWidgets('can filter and select a list of custom User options', (WidgetTester tester) async {
+  testWidgets('can filter and select a list of custom User options',
```

Comment:
There are still more unrelated changes in this PR that need to be reverted. This is happening throughout the `autocomplete_test.dart` file probably due to the use of `dartfmt`.

Reply:
Ohh okay let  me take a look.
Sorry for all the mess

Reply:
Can you please review now.